synfig-studio/src/synfigapp/canvasinterface.cpp

```diff
@@ -941,51 +943,56 @@ CanvasInterface::import(
 
 bool
 CanvasInterface::import_sequence(
-	const std::set<synfig::String> &filenames,
-	synfig::String &errors,
-	synfig::String &/*warnings*/,
-	bool resize_image )
+		const std::set<synfig::String> &filenames,
+		synfig::String &errors,
+		synfig::String &/*warnings*/,
+		bool resize_image,
+		UIInterface::Response rem_dups)
```

Comment:
I used `yes_no_cancel()` for the user preference on importing duplicates. It returns a `UIInterface::Response`. So I thought it will be convenient this way. Also, `import_sequence()` is used only once. I changed it `int` now.